app/src/main/java/mozilla/org/webmaker/javascript/WebAppInterface.java

```diff
@@ -29,6 +29,9 @@ public WebAppInterface(Context c, JSONObject routeParams) {
         mPrefs = mContext.getSharedPreferences(mPrefKey, 0);
         mPageState = mPrefs.getString("page_state", "{}");
         mRoute = routeParams;
+        if (routeParams != null) {
```

Comment:
Probably better to log this in `WebmakerActivity` where the object is first created instead.